src/pages/new-claim/new-claim.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+import { useParams } from 'react-router';
+import { Redirect } from 'react-router-dom';
+import { BaseLayout } from '../../components/layout';
+import Button from '../../components/button';
+import NewClaimForm, { FormState, FormStatus, parseClaimAmount } from './new-claim-form';
+import { useClaimsManager } from '../../contracts';
+import { useUserPolicyById, canCreateClaim } from '../../logic/policies';
+import globalStyles from '../../styles/global-styles.module.scss';
+import styles from './new-claim.module.scss';
+
+interface Params {
+  policyId: string;
+}
+
+export default function NewClaim() {
+  const { policyId } = useParams<Params>();
+  const [step, setStep] = useState<'instructions' | 'capture' | 'confirmation'>('instructions');
+
+  const claimsManager = useClaimsManager();
+  const { data: policy, status: loadStatus } = useUserPolicyById(policyId);
+
+  const [form, setForm] = useState<FormState>({ evidence: '', amount: '' });
+  const [status, setStatus] = useState<FormStatus>('idle');
+
+  if (!claimsManager) {
+    return (
+      <BaseLayout subtitle="New Claim">
+        <p className={globalStyles.textCenter}>Please connect your wallet to submit a claim.</p>
+      </BaseLayout>
+    );
+  }
+
+  if (!policy) {
+    return (
+      <BaseLayout subtitle="New Claim">
+        <h4 className={styles.heading}>New Claim</h4>
+        {loadStatus === 'loading' && <p className={globalStyles.secondaryColor}>Loading...</p>}
+        {loadStatus === 'loaded' && <p>Unable to find your policy with given id.</p>}
+      </BaseLayout>
+    );
+  }
+
+  if (!canCreateClaim(policy)) {
+    return <Redirect to={'/policies/' + policy.policyId} />;
+  }
+
+  const handleSubmit = async () => {
+    setStatus('submitting');
+    try {
+      await claimsManager.createClaim(
+        policy.beneficiary,
+        policy.coverageAmount,
+        Math.round(policy.startTime.getTime() / 1000),
+        Math.round(policy.endTime.getTime() / 1000),
+        policy.ipfsHash,
+        parseClaimAmount(form.amount),
+        form.evidence.trim()
+      );
+
+      setStatus('submitted');
+    } catch (err) {
+      setStatus('failed');
+    }
+  };
+
+  if (status === 'submitted') {
+    return (
+      <BaseLayout subtitle="New Claim">
+        <p>Success TODO</p>
```

Comment:
I see some TODOs in the file - I guess this is because the labels are not yet finalized, right?

Reply:
Hmm I'm not sure how, but you ended up commenting on a specific commit (and not the actual PR): https://github.com/api3dao/api3-dao-dashboard/pull/299/commits/81917d4dddb26acfd088bf5eb6f01a5767c9f804

I think this happened in the previous PR as well. Do you know how this happens as it's probably not where you meant to review?